src/Toml/Bi/Map.hs

```diff
@@ -460,6 +462,29 @@ _LByteStringText = prism (TL.encodeUtf8 . TL.fromStrict) eitherText
 _LByteString :: TomlBiMap BL.ByteString AnyValue
 _LByteString = _LByteStringText >>> _Text
 
+-- | Helper bimap between a Word8 lists and Int lists
+_Word8ArrIntArr :: TomlBiMap [Word8] [Int]
+_Word8ArrIntArr = BiMap
+    { forward  = Right . map fromIntegral
+    , backward = eitherNonNeg
+    }
+  where
+    eitherNonNeg :: [Int] -> Either TomlBiMapError [Word8]
+    eitherNonNeg xs
+      | all (>= 0) xs = Right $ map fromIntegral xs
+      | otherwise     = Left $
+          ArbitraryError ("Negative numbers not allowed when converting "
+              <> tShow xs
+              <> " to a ByteString")
+
+-- | Helper bimap for a 'Integer' list and strict 'ByteString'
+_ByteStringArray :: TomlBiMap ByteString [Int]
+_ByteStringArray = iso unpack pack >>> _Word8ArrIntArr
```

Comment:
Let's instead add `import qualified Data.ByteString as BS` and use qualified `unpack` and `pack` here. For code readability.